packages/driver/test/cypress/integration/cypress/cypress_spec.coffee

```diff
@@ -45,8 +45,6 @@ describe "driver/src/cypress/index", ->
         expect(err.backend).to.be.true
         expect(err.stack).not.to.include("From previous event")
 
```

Comment:
this needed a `done` callback because there's no guarantee the `.catch` handler was fired - so we either need to use `done` or use a `.then(() => throw new Error(...)` or `expect(@Cypress.backend("foo")).to.eventually.be.rejectedWith(...)`